common/src/main/java/io/quarkus/ts/openshift/common/OpenShiftTestExtension.java

```diff
@@ -450,6 +455,15 @@ public void handleTestExecutionException(ExtensionContext context, Throwable thr
         throw throwable;
     }
 
+    private void runOnFailureAction(ExtensionContext context, OnOpenShiftFailureAction action) {
+        try {
+            postProcessTestInstance(action, context);
```

Comment:
This looks weird on the first sight, we should extract the injection logic into a separate, properly named method, so that it's clear what we do here.

Reply:
I agree with you, but we cannot change the postProcessTestInstance name... I can create another proper named method and make the postProcessTestInstance and runOnFailureAction to call to the new one if you prefer so.

Reply:
That's what I meant :-)

Reply:
Changed.